binary-search.html.markdown

```diff
@@ -3,6 +3,7 @@ category: Algorithms & Data Structures
 name: Binary Search
 contributors:
     - ["Abhishek Jaisingh", "http://github.com/abhishekjiitr"]
+    - ["Elton Viana", "http://github.com/eltonvs"]
```

Comment:
I'm sorry, but we reserve the contributor tag for more significant contributions